proto/algorithm_state.proto

```diff
@@ -41,8 +42,9 @@ message AlgorithmState {
       NIGDistribution nnig_state = 2;
       NWDistribution nnw_state = 3;
       MultiNormalIGDistribution lin_reg_uni_state = 4;
+      MFAPriorDistribution mfa_state = 5; //TODO change name
     }
   }
-  HierarchyHypers hierarchy_hypers = 5; // The current values of the hyperparameters of the hierarchy
+  HierarchyHypers hierarchy_hypers = 6; // The current values of the hyperparameters of the hierarchy
```

Comment:
I believe we should not change the current numbering, as it could break backwards compatibility or something. Is this correct @mberaha?

Reply:
Yes you are totally right, please change back to the original numbering and let `mfa_state = 6;`. Maybe add a // TODO to change the numbering in a future release